app/assets/javascripts/duties.js

```diff
@@ -87,10 +124,12 @@ function setDutyTableWidth(averageColspan) {
 
 function load() {
   var START_TIME = $('#duty-start-time').data('start-time');
+  var END_TIME = 21;//I need to hard code end time because I can't find the end time in database...
```

Comment:
There's a way to make this not using hard-code. Check line 1 of `/app/views/duties/index.html.erb`, it grabs the start time from the database, contain it in a data attribute of a html tag, and access it on javascript side on line 126 above. Can you make it like that as well?